.travis.yml

```diff
@@ -9,8 +9,11 @@ install:
 script:
   - mkdir -p build/logs
   - cd tests
-  - phpunit --coverage-clover ../build/logs/clover.xml .
+  - phpunit --coverage-clover=coverage.clover ../build/logs/clover.xml .
+  - wget https://scrutinizer-ci.com/ocular.phar
+  - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
```

Comment:
The `coverage.clover` needs to be replaced with `../build/logs/clover.xml` because it's filename to file, that has Coverage Info in Clover format.